content/docs/news/release_announcements/keptn-100/index.md

```diff
@@ -0,0 +1,124 @@
+* **installer:** Remove duplicate volumes and volumeMounts configuration ([#8949](https://github.com/keptn/keptn/issues/8949)) ([5f034c7](https://github.com/keptn/keptn/commit/5f034c700fa281260588f5ba41b6956e6f8fbebc))
+* **lighthouse-service:** don't fail sequence if SLO file is missing ([#9153](https://github.com/keptn/keptn/issues/9153)) ([6b1f05f](https://github.com/keptn/keptn/commit/6b1f05fcf547a4863f5f1b76cef7dc9e37d2b920))
+* **lighthouse-service:** fail with descriptive error message when SLO criteria and target cannot be parsed ([#9206](https://github.com/keptn/keptn/issues/9206)) ([56977a3](https://github.com/keptn/keptn/commit/56977a3ce1cbd2f5eda4de003a8356f404d641af))
+* **lighthouse-service:** handling no SLO objectives ([#9203](https://github.com/keptn/keptn/issues/9203)) ([5626961](https://github.com/keptn/keptn/commit/56269619102da634e87816b13b8faca051040bc3))
+* **lighthouse-service:** Include SLO Display Name also when there is no SLI value available ([#9194](https://github.com/keptn/keptn/issues/9194)) ([9ebed1e](https://github.com/keptn/keptn/commit/9ebed1efbc48d0a6b0725cb8884da462bdbda418))
+* **lighthouse-service:** Return error if slo.yaml cannot be fetched from resource service ([#9143](https://github.com/keptn/keptn/issues/9143)) ([723ba6d](https://github.com/keptn/keptn/commit/723ba6d0f750a32210d7c7b2dcb7c2da00f56c58))
+* Resolve security scan issues ([#9094](https://github.com/keptn/keptn/issues/9094)) ([d50bfde](https://github.com/keptn/keptn/commit/d50bfde5cb37ee26fecbdac5d508681f61c6be9a))
+* **resource-service:** Delete tmp-origin before migrating repository ([#9104](https://github.com/keptn/keptn/issues/9104)) ([b3e368f](https://github.com/keptn/keptn/commit/b3e368fc2df93d8f25ff9a1f3f324586cb1c948f))
+* **resource-service:** Delete tmp-origin before migrating repository ([#9106](https://github.com/keptn/keptn/issues/9106)) ([b3e368f](https://github.com/keptn/keptn/commit/b3e368fc2df93d8f25ff9a1f3f324586cb1c948f))
+* **resource-service:** Determine default branch from helm value when repository is not initialized ([#8843](https://github.com/keptn/keptn/issues/8843)) ([8e91639](https://github.com/keptn/keptn/commit/8e916394247c957c9f029e651f6d345315b37979))
+* **resource-service:** Force checkout event if unstaged changes are present ([#9107](https://github.com/keptn/keptn/issues/9107)) ([dbd0ddb](https://github.com/keptn/keptn/commit/dbd0ddba017001f4864a40b6a26cf867614ea1dc))
+* **resource-service:** Map go-git specific error types to Keptn error types understood by the resource service error handler ([#8849](https://github.com/keptn/keptn/issues/8849)) ([75a1314](https://github.com/keptn/keptn/commit/75a1314c1fbfab6040346b98bfb8e13331a7c460))
+* **resource-service:** Return specific error in when creating a project with an initialized repository ([#8855](https://github.com/keptn/keptn/issues/8855)) ([2f7e1f2](https://github.com/keptn/keptn/commit/2f7e1f273ab0df258fac3a493d4fde9947b86f18))
```

Comment:
"...error in when creating" isn't right -- not sure if you are missing a word after "in" or if "in" should be removed.